.github/workflows/docker-deploy.yaml

```diff
@@ -0,0 +1,31 @@
+name: docker-deploy
+
+on: 
+  push:
+    branches: 
+    - master
+
+jobs:
+  deploy:
+    name: Build and Push Docker
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout JETSCAPE
+      uses: actions/checkout@v2
+      with:
+        path: JETSCAPE-DOCKER
+        
+    - name: Copy Dockerfile
+      run: cp JETSCAPE-DOCKER/docker/Dockerfile-github Dockerfile
+      
+    #https://github.com/marketplace/actions/docker-publish
+    #https://www.docker.com/blog/first-docker-github-action-is-here/
+    - name: Dockerize
+      uses: docker/build-push-action@v1
+      with:
+       username: ${{ secrets.DOCKER_USERNAME }}
+       password: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
The secrets need to be updated for the Jetscape repository to set to dockerhub repository: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets

Reply:
Done